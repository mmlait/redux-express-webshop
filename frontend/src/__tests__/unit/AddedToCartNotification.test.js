import React from 'react';
import { create } from 'react-test-renderer'
import AddedToCartNotification from '../../components/AddedToCartNotification';

describe('AddedToCartNotification component', () => {
    it('should render', () => {
    let tree = create(<AddedToCartNotification />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})