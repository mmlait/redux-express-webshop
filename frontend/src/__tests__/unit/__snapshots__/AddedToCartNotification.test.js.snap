// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddedToCartNotification component should render 1`] = `
<div
  className="sc-bdVaJa gtzOcq"
  data-test="modal"
>
  <p>
    Product added to cart!
  </p>
</div>
`;
